src/module.ts

```diff
@@ -124,7 +124,14 @@ export default defineNuxtModule({
         _viewerDevMiddleware(req, res)
       }
       addServerMiddleware({ path, handler: viewerDevMiddleware })
-      nuxt.hook('listen', () => { logger.info(`Tailwind Viewer: ${chalk.underline.yellow(path)}`) })
+      nuxt.hook('listen', (_, listener) => {
+        const url = withTrailingSlash(joinURL(listener.url, path))
+        if (isNuxt2() && !(nuxt.options as any).bridge) {
+          nuxt.options.cli.badgeMessages.push(`Tailwind Viewer: ${chalk.underline.yellow(url)}`)
```

Comment:
The bridge should be used with `nuxi` CLI which has no support of `badgeMessages`. This change doesn't make sense for v5